src/commons/services/utils.js

```diff
@@ -108,6 +108,35 @@
             return clusterName;
         }
 
+        vm.getVolumeDetails = function(clusterId) {
+            volumeList = [];
+            clusterData = vm.clusterData.clusters;
+            len = clusterData.length;
+            for ( i = 0; i < len; i++ ) {
+                clusterObj = clusterData[i];
+                for( key in clusterObj ) {
```

Comment:
@kamleshverma1 Agreed. In that case you should check for "tendrl_context" property instead of checking "not equal to stats" as in the future it may be possible that there can be more properties other than stats too.